W1-M2-numbers-in-csharp-local/numbers-quickstart/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿double third = 1.0 / 3.0;

```

Comment:
Notice with dotnet 6 how void main() {} has dissapeared. In dotnet 6, Microsoft decided to remove a bunch of code you used to need to write to make things work. This code, considered decorative is now hidden (although still works) - However it is important to understand what is going on behind the scenes (we do a lot of behind the scenes stuff) - but that will come